### ISSUE TITLE
models.py:7-9: Continue implementation. Table Shape...

### ISSUE THREAD

**Issue by 0pdd** [timelesslounge/timelessis#72]
The puzzle `13-e94c1888` from #13 has to be resolved: 

https://github.com/timelesslounge/timelessis/blob/664b147135ef22415eb5f876b8762e33123cc80b/restaurants/models.py#L7-L9

The puzzle was created by @amihaiemil on 29-Jan-19. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by krzyk** [comment]
@0crat waiting for review of PR #136

**Comment by 0crat** [comment]
> @0crat waiting for review of PR #136  ([here](https://github.com/timelesslounge/timelessis/issues/72#issuecomment-461010655))

@krzyk There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@55d69b91-ec34-4bf1-baee-b9c4b56fbdad, thread: AsyncProc-1
com.zerocracy.farm.sync.SyncProject[76] java.lang.IllegalStateException: Failed to acquire "impediments.xml" in "CF9LG1YCA" in 2min: java.util.concurrent.locks.ReentrantLock@62e21b4b[Locked by thread BkParallel-7]
```

[0.40.15](https://github.com/zerocracy/farm): CID: [eb53b1ba-791b-4858-af34-acb68958a801](https://www.0crat.com/footprint/CF9LG1YCA/eb53b1ba-791b-4858-af34-acb68958a801), Type: "Register impediment"

**Comment by krzyk** [comment]
@0crat waiting for review of PR #136

**Comment by 0crat** [comment]
> @0crat waiting for review of PR #136 ([here](https://github.com/timelesslounge/timelessis/issues/72#issuecomment-461057420))

@krzyk The impediment for [#72](https://github.com/timelesslounge/timelessis/issues/72) was registered successfully by @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by emilianodellacasa** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/timelesslounge/timelessis/issues/72#issuecomment-461867282))

@emilianodellacasa This is what I know about this job in [CF9LG1YCA](https://www.0crat.com/p/CF9LG1YCA), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#72](https://github.com/timelesslounge/timelessis/issues/72) is [in scope](http://www.zerocracy.com/policy.html#14) for 9days
  * The role is `DEV`
  * The job is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk) for [8days](http://www.zerocracy.com/policy.html#8)
  * There is a monetary reward attached
  * @krzyk[/z](https://www.0crat.com/u/krzyk) will get 50% of their hourly rate for this job
  * The job has an [impediment](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 30 minutes/points
  * These users are banned and won't be assigned:
    * @0pdd[/z](https://www.0crat.com/u/0pdd): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/CF9LG1YCA?q=%7Bjob%3A%27gh%3Atimelesslounge%2Ftimelessis%2372%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by 0pdd** [comment]
The puzzle `13-e94c1888` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#162](https://github.com/timelesslounge/timelessis/issues/162) is still not solved.

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#72](https://github.com/timelesslounge/timelessis/issues/72) is now out of scope

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0pdd** [comment]
@0pdd 4 puzzles [#203](https://github.com/timelesslounge/timelessis/issues/203), [#204](https://github.com/timelesslounge/timelessis/issues/204), [#205](https://github.com/timelesslounge/timelessis/issues/205), [#206](https://github.com/timelesslounge/timelessis/issues/206) are still not solved; solved: [#162](https://github.com/timelesslounge/timelessis/issues/162).

**Comment by 0pdd** [comment]
@0pdd 4 puzzles [#204](https://github.com/timelesslounge/timelessis/issues/204), [#205](https://github.com/timelesslounge/timelessis/issues/205), [#206](https://github.com/timelesslounge/timelessis/issues/206), [#260](https://github.com/timelesslounge/timelessis/issues/260) are still not solved; solved: [#162](https://github.com/timelesslounge/timelessis/issues/162), [#203](https://github.com/timelesslounge/timelessis/issues/203).

**Comment by 0pdd** [comment]
@0pdd 5 puzzles [#204](https://github.com/timelesslounge/timelessis/issues/204), [#205](https://github.com/timelesslounge/timelessis/issues/205), [#260](https://github.com/timelesslounge/timelessis/issues/260), [#285](https://github.com/timelesslounge/timelessis/issues/285), [#286](https://github.com/timelesslounge/timelessis/issues/286) are still not solved; solved: [#162](https://github.com/timelesslounge/timelessis/issues/162), [#203](https://github.com/timelesslounge/timelessis/issues/203), [#206](https://github.com/timelesslounge/timelessis/issues/206).

**Comment by 0pdd** [comment]
@0pdd 8 puzzles [#204](https://github.com/timelesslounge/timelessis/issues/204), [#205](https://github.com/timelesslounge/timelessis/issues/205), [#285](https://github.com/timelesslounge/timelessis/issues/285), [#286](https://github.com/timelesslounge/timelessis/issues/286), [#314](https://github.com/timelesslounge/timelessis/issues/314), [#315](https://github.com/timelesslounge/timelessis/issues/315), [#316](https://github.com/timelesslounge/timelessis/issues/316), [#317](https://github.com/timelesslounge/timelessis/issues/317) are still not solved; solved: [#162](https://github.com/timelesslounge/timelessis/issues/162), [#203](https://github.com/timelesslounge/timelessis/issues/203), [#206](https://github.com/timelesslounge/timelessis/issues/206), [#260](https://github.com/timelesslounge/timelessis/issues/260).

**Comment by 0pdd** [comment]
@0pdd 8 puzzles [#205](https://github.com/timelesslounge/timelessis/issues/205), [#285](https://github.com/timelesslounge/timelessis/issues/285), [#286](https://github.com/timelesslounge/timelessis/issues/286), [#314](https://github.com/timelesslounge/timelessis/issues/314), [#315](https://github.com/timelesslounge/timelessis/issues/315), [#316](https://github.com/timelesslounge/timelessis/issues/316), [#317](https://github.com/timelesslounge/timelessis/issues/317), [#354](https://github.com/timelesslounge/timelessis/issues/354) are still not solved; solved: [#162](https://github.com/timelesslounge/timelessis/issues/162), [#203](https://github.com/timelesslounge/timelessis/issues/203), [#204](https://github.com/timelesslounge/timelessis/issues/204), [#206](https://github.com/timelesslounge/timelessis/issues/206), [#260](https://github.com/timelesslounge/timelessis/issues/260).

**Comment by 0pdd** [comment]
@0pdd 10 puzzles [#285](https://github.com/timelesslounge/timelessis/issues/285), [#286](https://github.com/timelesslounge/timelessis/issues/286), [#314](https://github.com/timelesslounge/timelessis/issues/314), [#315](https://github.com/timelesslounge/timelessis/issues/315), [#316](https://github.com/timelesslounge/timelessis/issues/316), [#317](https://github.com/timelesslounge/timelessis/issues/317), [#354](https://github.com/timelesslounge/timelessis/issues/354), [#414](https://github.com/timelesslounge/timelessis/issues/414), [#415](https://github.com/timelesslounge/timelessis/issues/415), [#416](https://github.com/timelesslounge/timelessis/issues/416) are still not solved; solved: [#162](https://github.com/timelesslounge/timelessis/issues/162), [#203](https://github.com/timelesslounge/timelessis/issues/203), [#204](https://github.com/timelesslounge/timelessis/issues/204), [#205](https://github.com/timelesslounge/timelessis/issues/205), [#206](https://github.com/timelesslounge/timelessis/issues/206), [#260](https://github.com/timelesslounge/timelessis/issues/260).

**Comment by 0pdd** [comment]
@0pdd 10 puzzles [#205](https://github.com/timelesslounge/timelessis/issues/205), [#285](https://github.com/timelesslounge/timelessis/issues/285), [#286](https://github.com/timelesslounge/timelessis/issues/286), [#314](https://github.com/timelesslounge/timelessis/issues/314), [#315](https://github.com/timelesslounge/timelessis/issues/315), [#316](https://github.com/timelesslounge/timelessis/issues/316), [#317](https://github.com/timelesslounge/timelessis/issues/317), [#354](https://github.com/timelesslounge/timelessis/issues/354), [#414](https://github.com/timelesslounge/timelessis/issues/414), [#416](https://github.com/timelesslounge/timelessis/issues/416) are still not solved; solved: [#162](https://github.com/timelesslounge/timelessis/issues/162), [#203](https://github.com/timelesslounge/timelessis/issues/203), [#204](https://github.com/timelesslounge/timelessis/issues/204), [#206](https://github.com/timelesslounge/timelessis/issues/206), [#260](https://github.com/timelesslounge/timelessis/issues/260), [#415](https://github.com/timelesslounge/timelessis/issues/415).

**Comment by 0pdd** [comment]
@0pdd 10 puzzles [#205](https://github.com/timelesslounge/timelessis/issues/205), [#286](https://github.com/timelesslounge/timelessis/issues/286), [#314](https://github.com/timelesslounge/timelessis/issues/314), [#315](https://github.com/timelesslounge/timelessis/issues/315), [#316](https://github.com/timelesslounge/timelessis/issues/316), [#317](https://github.com/timelesslounge/timelessis/issues/317), [#354](https://github.com/timelesslounge/timelessis/issues/354), [#414](https://github.com/timelesslounge/timelessis/issues/414), [#416](https://github.com/timelesslounge/timelessis/issues/416), [#462](https://github.com/timelesslounge/timelessis/issues/462) are still not solved; solved: [#162](https://github.com/timelesslounge/timelessis/issues/162), [#203](https://github.com/timelesslounge/timelessis/issues/203), [#204](https://github.com/timelesslounge/timelessis/issues/204), [#206](https://github.com/timelesslounge/timelessis/issues/206), [#260](https://github.com/timelesslounge/timelessis/issues/260), [#285](https://github.com/timelesslounge/timelessis/issues/285), [#415](https://github.com/timelesslounge/timelessis/issues/415).

**Comment by 0pdd** [comment]
@0pdd 9 puzzles [#286](https://github.com/timelesslounge/timelessis/issues/286), [#314](https://github.com/timelesslounge/timelessis/issues/314), [#315](https://github.com/timelesslounge/timelessis/issues/315), [#316](https://github.com/timelesslounge/timelessis/issues/316), [#317](https://github.com/timelesslounge/timelessis/issues/317), [#354](https://github.com/timelesslounge/timelessis/issues/354), [#414](https://github.com/timelesslounge/timelessis/issues/414), [#416](https://github.com/timelesslounge/timelessis/issues/416), [#462](https://github.com/timelesslounge/timelessis/issues/462) are still not solved; solved: [#162](https://github.com/timelesslounge/timelessis/issues/162), [#203](https://github.com/timelesslounge/timelessis/issues/203), [#204](https://github.com/timelesslounge/timelessis/issues/204), [#205](https://github.com/timelesslounge/timelessis/issues/205), [#206](https://github.com/timelesslounge/timelessis/issues/206), [#260](https://github.com/timelesslounge/timelessis/issues/260), [#285](https://github.com/timelesslounge/timelessis/issues/285), [#415](https://github.com/timelesslounge/timelessis/issues/415).

**Comment by 0pdd** [comment]
@0pdd 8 puzzles [#286](https://github.com/timelesslounge/timelessis/issues/286), [#314](https://github.com/timelesslounge/timelessis/issues/314), [#316](https://github.com/timelesslounge/timelessis/issues/316), [#317](https://github.com/timelesslounge/timelessis/issues/317), [#354](https://github.com/timelesslounge/timelessis/issues/354), [#414](https://github.com/timelesslounge/timelessis/issues/414), [#416](https://github.com/timelesslounge/timelessis/issues/416), [#462](https://github.com/timelesslounge/timelessis/issues/462) are still not solved; solved: [#162](https://github.com/timelesslounge/timelessis/issues/162), [#203](https://github.com/timelesslounge/timelessis/issues/203), [#204](https://github.com/timelesslounge/timelessis/issues/204), [#205](https://github.com/timelesslounge/timelessis/issues/205), [#206](https://github.com/timelesslounge/timelessis/issues/206), [#260](https://github.com/timelesslounge/timelessis/issues/260), [#285](https://github.com/timelesslounge/timelessis/issues/285), [#315](https://github.com/timelesslounge/timelessis/issues/315), [#415](https://github.com/timelesslounge/timelessis/issues/415).

**Comment by 0pdd** [comment]
@0pdd 8 puzzles [#286](https://github.com/timelesslounge/timelessis/issues/286), [#314](https://github.com/timelesslounge/timelessis/issues/314), [#317](https://github.com/timelesslounge/timelessis/issues/317), [#354](https://github.com/timelesslounge/timelessis/issues/354), [#414](https://github.com/timelesslounge/timelessis/issues/414), [#416](https://github.com/timelesslounge/timelessis/issues/416), [#462](https://github.com/timelesslounge/timelessis/issues/462), [#538](https://github.com/timelesslounge/timelessis/issues/538) are still not solved; solved: [#162](https://github.com/timelesslounge/timelessis/issues/162), [#203](https://github.com/timelesslounge/timelessis/issues/203), [#204](https://github.com/timelesslounge/timelessis/issues/204), [#205](https://github.com/timelesslounge/timelessis/issues/205), [#206](https://github.com/timelesslounge/timelessis/issues/206), [#260](https://github.com/timelesslounge/timelessis/issues/260), [#285](https://github.com/timelesslounge/timelessis/issues/285), [#315](https://github.com/timelesslounge/timelessis/issues/315), [#316](https://github.com/timelesslounge/timelessis/issues/316), [#415](https://github.com/timelesslounge/timelessis/issues/415).
